js/views/conversation_view.js

```diff
@@ -938,6 +945,7 @@
                     convo.sendMessage(message, attachments);
                 });
                 input.val("");
+                this.focusMessageField();
```

Comment:
To be sure that message sending is synchronized WRT the contents of the message text area, should this call be moved *within* the promise above, e.g.

```
this.fileInput.getFiles().then(function(attachments) {
    convo.sendMessage(message, attachments);
    this.focusMessageField();
}).catch(function(error) {
    console.log('foobar');
    this.focusMessageField();
});
```

Otherwise this seems redundant or maybe nondeterministic with the other call to `this.focusMessageField()` on (old) line 934?